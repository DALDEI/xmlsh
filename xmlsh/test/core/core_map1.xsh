# Test of basic map features
a={}
xtype $a
a+={ name : value }
xtype $a
a+={ name2 : "value 2" }
echo ${a[name2]}
b={ $a , "c" : { "a1" : "value1" , "a2" : value2 } }
xtype $b
echo ${b[c]}