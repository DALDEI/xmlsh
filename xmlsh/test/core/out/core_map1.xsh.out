org.xmlsh.core.XValueProperty
value 2
org.xmlsh.core.XValuePropertyList
[value1 ,value2 ]
